user_docs/en/userGuide.t2t

```diff
@@ -2589,33 +2589,67 @@ To toggle the braille viewer from anywhere, please assign a custom gesture using
+
+Selecting an add-on will display the details for the add-on in the next panel.
+Associated actions will also become available for the add-on: e.g. to install, update, disable, enable or remove the add-on.
+If there is help available for an installed add-on, you can access it by pressing the "Add-on help" button (``alt+h``).
+
+++++ Filter add-ons by status ++++[AddonStoreFilterStatus]
+To change the view of add-ons, change the "Filter by Status" (``alt+f``).
+You can filter by installed, updatable, disabled or available add-ons.
+To include outdated [incompatible add-ons #incompatibleAddonsManager] in available add-ons, enable the [relevant setting #IncompatibleAddonsSetting] in NVDA preferences.
+
+++++ Filter add-ons by channel ++++[AddonStoreFilterChannel]
+To change the selection of add-ons, change the "Channel" filter selection (``alt+n``).
+
+There are 3 channels of add-ons:
+- Stable: The developer has released this as a tested add-on with a released version of NVDA.
+- Beta: This add-on may need further testing.
+Suggested for early adopters.
+- Dev: This channel is suggested to be used by add-on developers to test unreleased API changes.
+NVDA alpha and beta testers may need to use a "Dev" version of their add-ons.
+-
+
+++++ Search add-ons ++++[AddonStoreFilterSearch]
+Add-ons can be filtered by display name, publisher and description.
+To search add-ons, use the "Search" text box (``alt+s``).
+
++++ Installing add-ons +++[AddonStoreInstalling]
+It is very important to only install add-ons from sources you trust.
+The functionality of add-ons is unrestricted inside NVDA. 
+This could include accessing your personal data or even the entire system.
+
+You can install and update add-ons by [browsing the Add-on Store #AddonStoreBrowsing].
+
+To install an add-on you have obtained outside of the Add-on Store, press the "Install from external source" button (``alt+i``).
+This will allow you to browse for an add-on package (``.nvda-addon`` file) somewhere on your computer or on a network.
 Once you press Open, the installation process will begin.
 
-When an add-on is being installed, NVDA will first ask you to confirm that you really wish to install the add-on.
-As the functionality of add-ons is unrestricted inside NVDA, which in theory could include accessing your personal data or even the entire system if NVDA is an installed copy, it is very important to only install add-ons from sources you trust.
+If NVDA is installed and running on your system, you can also open a add-on file directly from the browser to begin the installation process.
+
+When an add-on is being installed, NVDA will ask you to confirm the installation.
 Once the add-on is installed, NVDA must be restarted for the add-on to start running. 
-Until you do, a status of "install" will show for that add-on in the list.
 
++++ Removing Add-ons +++[RemovingAddonStore]
 To remove an add-on, select the add-on from the list and press the Remove button.

```

Comment:
Would it be worth saying "...select the add-on from the list and activate the Remove button (alt+r)."